dbt/macros/_get_final_acquisition_status_list.yml

```
@@ -0,0 +1,10 @@
+version: 2
+
+macros:
+  - name: get_final_acquisition_status_list
+    description: "Retourne la liste des statuts finaux possibles pour l'acquisition"
+    config:
+      docs:
+        show: true
+      meta: {}
+    arguments: []
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)